### PR TITLE
Fixes the route for write dashboard

### PR DESCRIPTION
**Pull request by cyriltovena**

### CODE REVIEW

**Review by kolesnikovae — APPROVED** [review]
LGTM

**Review by simonswine — APPROVED** [review]
LGTM

File: operations/phlare/jsonnet/phlare-mixin/phlare-mixin/dashboards/phlare-writes.libsonnet [hunk 1]
```diff
@@ -60,7 +60,7 @@ local utils = import 'mixin-utils/utils.libsonnet';
                             $.row('Distributor Requests')
                             .addPanel(
                               $.panel('QPS') +
-                              $.qpsPanel('pyroscope_request_duration_seconds_count{%s, route=~".*push.*"}' % std.rstripChars(dashboards['phlare-writes.json'].distributorSelector, ','))
+                              $.qpsPanel('pyroscope_request_duration_seconds_count{%s, route=~".*push.*|ingest"}' % std.rstripChars(dashboards['phlare-writes.json'].distributorSelector, ','))
```
[the remark this answers is not in the document]
Comment (cyriltovena) [reply operations/phlare/jsonnet/phlare-mixin/phlare-mixin/dashboards/phlare-writes.libsonnet:63]: hum I can wildcard it.

[the remark this answers is not in the document]
Comment (cyriltovena) [reply operations/phlare/jsonnet/phlare-mixin/phlare-mixin/dashboards/phlare-writes.libsonnet:63]: shoot automerge

**Pull request merged**